Sources/SpeziLLMOpenAI/Onboarding/LLMOpenAIModelOnboardingStep.swift

```diff
@@ -14,14 +14,27 @@ import SwiftUI
 /// View to display an onboarding step for the user to enter change the OpenAI model.
 public struct LLMOpenAIModelOnboardingStep: View {
     public enum Default {
-        public static let models: [LLMOpenAIModelType] = [.gpt3_5Turbo, .gpt4_turbo, .gpt4_o]
+        public static let models: [LLMOpenAIRequestType.modelPayload] = [
+            .init(
+                value1: "GPT 3.5 Turbo",
```

Comment:
spelled `GPT-3.5 Turbo`

Reply:
Should I update that in the tests too? If yes, should that be a seperate PR or can I just do it here? 

For instance:
https://github.com/StanfordSpezi/SpeziLLM/blob/fd9f1cacb19199576656db4c4dd0c81923b3597a/Tests/UITests/TestAppUITests/TestAppLLMOpenAIUITests.swift#L101